tests/compute/credentials.go

```diff
@@ -128,11 +128,11 @@ var _ = SIGDescribe("Guest Access Credentials", func() {
 			"fedora": []byte(customPassword),
 		}
 
-		It("[test_id:6221]should propagate user password", func() {
+		It("[test_id:6221]should propagate user password", decorators.Conformance, func() {
```

Comment:
I'd vote against promoting this flavor of the test. It is too similar to test_id:6224 to merit a place in Conformance.

Reply:
Second look, looks more like `6220`, agree to drop it

Reply:
Done